package.json

```diff
@@ -39,5 +39,9 @@
       "path": "./node_modules/cz-conventional-changelog",
       "defaultScope": "global"
     }
+  },
+  "resolutions": {
+    "@oasis/client-gql": "portal:/Users/vrishank/Desktop/OasisDev/oasis-noapi/packages/client-gql",
+    "@oasis/web": "portal:/Users/vrishank/Desktop/OasisDev/oasis-noapi/packages/web"
```

Comment:
Does this need to be under a certain user (you in this case?)

Reply:
I think yarn link generated those, I'll fix that ig